templates/tips/categories.html

```diff
@@ -0,0 +1,43 @@
+{% extends "base.html" %} {%load static%} {%block content%}
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>HTML 5 Boilerplate</title>
+    <link rel="stylesheet" href="../../static/css/style.css">
+  </head>
+  <body>
+    <script src="index.js"></script>
+    {% for tip in category_filtered_tips %}
+        <article class="post vt-post">
+            <div class="row">
+                <div class="col-xs-12 col-sm-5 col-md-5 col-lg-4">
+                    <div class="post-type post-img">
+                        <a href="#"><img src="https://bootdey.com/img/Content/avatar/avatar1.png" class="img-responsive" alt="image post"></a>
+                    </div>
+                    <div class="author-info author-info-2">
+                        <ul class="list-inline">
+                            <li>
+                                <div class="info">
+                                    <p>Posted on:</p>
```

Comment:
We have added the option to edit a tip as well.
Is this date the Original Posting Date? Or the Last Modified Date?
I think the later would fit better :)

You could set the new date at the edit stage

Reply:
I actually think the first is better because, if the user wants to make a change in his tip like edit a typo for example
he doesn't neccesary want anyone to know about it.
What do you think @ronyyosef  @LiorKGOW  @Yoavsc129  @svatares

Reply:
Can I mark it as resolved?

Reply:
It doesn't have to say what was changed
Just that it was modified
What do you think @Yoavsc129 ?
We could go with just the posting date, I don't mind

Reply:
I don't think it's necessary to change it to last edit date